test/parallel/test-stringbytes-external.js

```diff
@@ -129,31 +129,31 @@ var PRE_3OF4_APEX = Math.ceil((EXTERN_APEX / 4) * 3) - RADIOS;
 
   assert.throws(function() {
     buf1.toString();
-  }, /toString failed|Invalid array buffer length/);
+  }, /toString\(\) failed: buffer larger|Invalid array buffer length/);
```

Comment:
I'm unsure if I should be included an error type here.

AFAIK this test is doing the exact same thing as buf1.toString('utf8')... in which case it would never throw with the message ```Invalid array buffer length```

If that is the case I should be able to get rid of the extra error message and check for RangeError